### PR TITLE
Adds workspace name to log downloads

### PR DESCRIPTION
**Pull request by krishnaglick**
## What
Adds workspace name to log downloads.

## How
Does a get workspace call and pulls out the name.

## Why
Brian had an ask after his ride with support, and it's a very simple addition.

### CODE REVIEW

File: airbyte-webapp/src/components/JobItem/components/DownloadButton.tsx [hunk 1]
```diff
@@ -15,12 +16,13 @@ interface DownloadButtonProps {
 
 const DownloadButton: React.FC<DownloadButtonProps> = ({ jobDebugInfo, fileName }) => {
   const { formatMessage } = useIntl();
+  const { name } = useGetWorkspace(useCurrentWorkspaceId());
```
Comment (krishnaglick) [inline airbyte-webapp/src/components/JobItem/components/DownloadButton.tsx:19]: We have a `useGetCloudWorkspace` that didn't seem to provide different behavior.

Comment (evantahler) [reply airbyte-webapp/src/components/JobItem/components/DownloadButton.tsx:19]: Does "name" have spaces and other junk in there?  do we have a function to "file-ize" the name?

```
// maybe?
const safeName = name.replace(/[^a-z0-9]/gi, '_').toLowerCase();
```

Comment (krishnaglick) [reply airbyte-webapp/src/components/JobItem/components/DownloadButton.tsx:19]: Good point! Went ahead and added something like this.

Comment (evantahler) [reply airbyte-webapp/src/components/JobItem/components/DownloadButton.tsx:19]: lol, `function fileize()` it is!

**Review by evantahler — APPROVED** [review]

**Pull request merged**